sale_order_line_service_view/wizard/wiz_delete_sale_line.py

```diff
@@ -14,6 +14,7 @@ class WizDeleteSaleLine(models.TransientModel):
 
     @api.model
     def default_get(self, var_fields):
+        super(WizDeleteSaleLine, self).default_get(var_fields)
```

Comment:
Aquí no deberías asignar a una variable y devolverla ya que haces el super?

Reply:
El default te devuelve un diccionario, y si se va a hacer super lo que habria que hacer es actualizarlo con lo que se hace en este código, aún así si se ha hecho únicamente por el pylint, este solo es un aviso.

Reply:
Ciertamente, al final también es un modelo que se genera en su módulo, y si hay algún módulo que lo herede o que lo vaya a heredar en el futuro será dependiente de este, pero bueno, si el travis solo da un warning y con eso se calla tampoco importaría mucho.

Reply:
@oihane , @Daniel-CA ... realizado lo que me habéis solicitado. Eskerrik asko.